### DISCUSSION TITLE
Disconnected cable

### DISCUSSION THREAD

**Discussion by juanmi513** [netprofilesmod/netprofilesmod#81] [Ideas]
When RJ45 is disconnected, profiles dissapear from the app and you can't select another until connect cable again.
I use it in a industrial enviroment, if i insert the cable in certain networks with the previous profile loaded, i can be fired lol.
I suggest profiles will be avalaible all the time, even if the cable is not connected to the card.

**Comment by geissbuehler** [comment]
Unfortunately this behaviour can't be changed as long as the IP settings are applied via WMI. I never have found an alternative solution that is simple enough for me to implement it ;) I assume that a proper solution would require to set the IPs it via Windows API calls. The issues #58 and #69 are already dealing with this problem. Unfortunately, no solution is in sight... Any help with this would be appreciated!

Cheers
Urs
